### ISSUE TITLE
[BUG] About the training process ！！

### ISSUE THREAD

**Issue by Z-Z-J** [mkocabas/VIBE#140]
Hello! I run the training code with the default config file you provided on the [mpii3d insta2d] training dataset ！！
![QQ截图20201019150152](https://user-images.githubusercontent.com/53818279/96412669-0f997980-121d-11eb-9f72-f58e057bd881.png)
![QQ截图20201019150313](https://user-images.githubusercontent.com/53818279/96412680-11633d00-121d-11eb-920d-214d680b1fc8.png)

Does the result look right?
Why does the training error increase(pa-mpjpe, mpjpe) ?  And the loss does not seem to decrease during training?

**Comment by Z-Z-J** [comment]
![QQ截图20201019152716](https://user-images.githubusercontent.com/53818279/96414835-36a57a80-1220-11eb-8ecd-47a78e896460.png)
![QQ截图20201019153416](https://user-images.githubusercontent.com/53818279/96415058-913ed680-1220-11eb-981b-fbe4faba16fc.png)

This result looks similar to yours !!
But, why does the training error increase(pa-mpjpe, mpjpe) ?  And the loss does not seem to decrease during training?

**Comment by bragilee** [comment]
Hi @Z-Z-J , May I know are you able to get full inst2d dataset? or you only use partial of insta2d as well.

**Comment by Z-Z-J** [comment]
> May I know are you able to get full inst2d dataset? or you only use partial of insta2d as well.

I just used the data provided by the author ！！
![QQ截图20201024093148](https://user-images.githubusercontent.com/53818279/97064844-c3af5180-15db-11eb-8783-d08c12d2565c.png)

**Comment by bragilee** [comment]
I see. Thank you so much.

> > May I know are you able to get full inst2d dataset? or you only use partial of insta2d as well.
> 
> I just used the data provided by the author ！！
> ![QQ截图20201024093148](https://user-images.githubusercontent.com/53818279/97064844-c3af5180-15db-11eb-8783-d08c12d2565c.png)

**Comment by lijiaman** [comment]
I also tried the training with their codes, got similar results. The PA-MPJPE got best performance in the first several epochs, then it didn't decrease... The loss curve looks similar, not decrease very obviously as expected..

**Comment by ZhangHedongTimes** [comment]
Have you solved the problem yet?

**Comment by Z-Z-J** [comment]
> 你解决这个问题了吗？

I haven't solved this problem so far

**Comment by halcyon370** [comment]
Same problem. Any solutions?

**Comment by jiandandian2** [comment]
The same problem, and I found the prepare_training data has not extract the 3dpw_train_db.pt code. Cannot use the 3dpw for training.

**Comment by jiandandian2** [comment]
The default training config had not 3dpw, after add 3dpw, the result became normal.
![Uploading 屏幕快照 2020-12-31 上午9.23.35.png…]()

**Comment by jiandandian2** [comment]
The default training config had not 3dpw, after add 3dpw, the result became normal.
<img width="1566" alt="屏幕快照 2020-12-31 上午9 23 35" src="https://user-images.githubusercontent.com/33227739/103389475-fc265b00-4b49-11eb-882f-8de6ef31531d.png">

**Comment by chriful** [comment]
> The default training config had not 3dpw, after add 3dpw, the result became normal.
> <img alt="屏幕快照 2020-12-31 上午9 23 35" width="1566" src="https://user-images.githubusercontent.com/33227739/103389475-fc265b00-4b49-11eb-882f-8de6ef31531d.png">

请问你是否解决了，用我们训练的模型运行的可视化结果比作者的模型看起来重建的人体比较小的情况

**Comment by halcyon370** [comment]
> 
> 
> > The default training config had not 3dpw, after add 3dpw, the result became normal.
> > <img alt="屏幕快照 2020-12-31 上午9 23 35" width="1566" src="https://user-images.githubusercontent.com/33227739/103389475-fc265b00-4b49-11eb-882f-8de6ef31531d.png">
> 
> 请问你是否解决了，用我们训练的模型运行的可视化结果比作者的模型看起来重建的人体比较小的情况

试试demo.py中设置bbox_scale=1.0？

**Comment by chriful** [comment]
> > > The default training config had not 3dpw, after add 3dpw, the result became normal.
> > > <img alt="屏幕快照 2020-12-31 上午9 23 35" width="1566" src="https://user-images.githubusercontent.com/33227739/103389475-fc265b00-4b49-11eb-882f-8de6ef31531d.png">
> > 
> > 
> > 请问你是否解决了，用我们训练的模型运行的可视化结果比作者的模型看起来重建的人体比较小的情况
> 
> 试试demo.py中设置bbox_scale=1.0？

非常感谢你，我的问题得到了解决！

**Comment by chriful** [comment]
> > > The default training config had not 3dpw, after add 3dpw, the result became normal.
> > > <img alt="屏幕快照 2020-12-31 上午9 23 35" width="1566" src="https://user-images.githubusercontent.com/33227739/103389475-fc265b00-4b49-11eb-882f-8de6ef31531d.png">
> > 
> > 
> > 请问你是否解决了，用我们训练的模型运行的可视化结果比作者的模型看起来重建的人体比较小的情况
> 
> 试试demo.py中设置bbox_scale=1.0？

非常感谢你，我的问题得到了解答！

**Comment by littlejiumi** [comment]
> > > > The default training config had not 3dpw, after add 3dpw, the result became normal.
> > > > <img alt="屏幕快照 2020-12-31 上午9 23 35" width="1566" src="https://user-images.githubusercontent.com/33227739/103389475-fc265b00-4b49-11eb-882f-8de6ef31531d.png">
> > > 
> > > 
> > > 请问你是否解决了，用我们训练的模型运行的可视化结果比作者的模型看起来重建的人体比较小的情况
> > 
> > 
> > 试试demo.py中设置bbox_scale=1.0？
> 
> 非常感谢你，我的问题得到了解答！

请问PoseTrack数据集下载2017的还是2018的呢？期待回复，谢谢～
